wallet-new/test/Integration/Expectations.hs

```diff
@@ -0,0 +1,29 @@
+module Integration.Expectations (MonadIntegrationExpectations(..)) where
+
+import           Universum
+
+import           Cardano.Wallet.Client.Http
+
+import           Integration.Util (Resp', asksM, fromResp)
+
+import qualified Test.Hspec as Hspec
+
+
+class MonadIntegrationExpectations m where
+    addressShouldExists :: Resp' WalletAddress -> m ()
+
+instance MonadIntegrationExpectations (ReaderT (WalletClient IO) IO) where
```

Comment:
`ReaderT` here only to make specs a bit more readable, which costs us a couple of `lift` and a `runSpec wc` call in front of each specs. 
Possibly, we could extend the `ReaderT` to also contain some sort of initial state with for instance, all the initial genesis wallet. This would prevent us from making extra requests to get those from the backend.